client_app/src/components/shelter/ShiftDetailsTable.js

```diff
@@ -1,24 +1,119 @@
 import React from "react";
-import Row from 'react-bootstrap/Row';
-import Col from 'react-bootstrap/Col';
 import "../../styles/shelter/ShiftDetailsTable.css";
 import shiftDetails from './ShiftDetailsData.tsx';
+import Tooltip from "@mui/material/Tooltip";
+import IconButton from "@mui/material/IconButton";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEnvelope, faCheck, faX, faPenToSquare, faTriangleExclamation} from '@fortawesome/free-solid-svg-icons';
+import { useNavigate } from "react-router-dom";
 
 export const ShiftDetailsTable = props => {
-  const { onClickMoreDetails } = props;
+  const navigate = useNavigate();
+  const { onSignUpVolunteersClick , onSendEmergencyAlertClick, onStatusModalClick} = props;
+
+  const emailButton = () => {
+    return (
+      <button className="emailButton" onClick={onSendEmergencyAlertClick}>
+        <FontAwesomeIcon icon={faEnvelope} size="1x" className="emailIcon" />
```

Comment:
Should we make it bigger?

Reply:
Yeah I haven't finished any of the styling